packages/framework/fluid-static/package.json

```diff
@@ -72,6 +72,15 @@
   },
   "typeValidation": {
     "version": "0.58.3000",
-    "broken": {}
+    "broken": {
+      "0.58.2000": {
+        "ClassDeclaration_FluidContainer": {
+          "forwardCompat": false
+        },
+        "InterfaceDeclaration_IFluidContainer": {
+          "forwardCompat": false
+        }
```

Comment:
@anthony-murphy @heliocliu, would this be considered a breaking change that needs to be merged into next instead of main? Initially, I thought it would be considered non-breaking to add `connectionState` to `IFluidContainer` and `FluidContainer`, but the new type validation showed that the previous version is not forward compatible. Thoughts on if this is still non-breaking and can be included in this PR?

Reply:
Forward incompatibility (probably) won't cause compile breaks, but you still need to consider if it will cause runtime compatibility breaks.  It looks like that won't be the case here since you're not calling into the loader layer changes from other layers, but it does mean you won't be able to use the new apis in main.  This is what Vlad was talking about in https://github.com/microsoft/FluidFramework/pull/9439#discussion_r824227574 as well

Reply:
I think this comment was referring to the new `connect()`/`disconnect()` functions being added to `Container` before adding them on `IContainer`. In this case, `connectionState` already exists on `Container`, so it should be able to work in main. Is that correct?

Reply:
yeah that will be fine